src/System.Security.Cryptography.Pkcs/tests/System.Security.Cryptography.Pkcs.Tests.csproj

```diff
@@ -48,6 +48,7 @@
     <Compile Include="SignedCms\SignedCmsTests.netcoreapp.cs" />
     <Compile Include="SignedCms\SignerInfoTests.netcoreapp.cs" />
     <Compile Include="Pkcs8PrivateKeyInfoTests.cs" />
+    <Compile Include="Pkcs9AttributeWindowsFileTimeTests.cs" />
```

Comment:
Why is this in the netcoreapp-only section? Especially if the point is to make sure it's the same between netfx and corefx I'd expect it to be not self-excluding from netfx.  (I'm not even sure why it's a new file, instead of just tests next to the existing value tests)

Reply:
My point here is to eliminate the CoreFX difference between windows and non-windows on the dates less than 1601. So I'm not sure that NetFX behavior can be aligned with this change too.
I observed failures of the same unit tests on NetFX target and assumed that it just includes some references like [this](https://github.com/dotnet/corefx/blob/master/src/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.csproj#L194-L197) and doesn't use existing source code. So I moved the related unit tests to separate netcoreapp-file. Please correct me if I'm wrong.

Reply:
Generally speaking, whatever .NET Framework does is the behavior we model after.  So if .NET Framework has the split ArgumentException/CryptographicException that CoreFx-Windows has, then the better fix is to make CoreFx-Unix match CoreFx-Windows.

On the other hand, if we choose to have "always CryptographicException" be the answer, we'd still want to have the test run under netfx and show that we've intentionally deviated the behavior. (We have an AssertExtensions method specifically for that, IIRC)

Either way, there shouldn't be a netcoreapp-specific file unless it is testing API that wasn't present in .NET Framework.

Reply:
Thanks for the explanation! I'd prefer to not bring some windows-specific check to unix part. Let me try the option to have CryptographicException on CoreFX and ArgumentOutOfRangeException on NetFX.